src/main/java/levi9/UserService/controller/UserController.java

```diff
@@ -0,0 +1,37 @@
+package levi9.UserService.controller;
+
+import levi9.UserService.dto.UserDto;
+import levi9.UserService.filter.SimpleAuthFilter;
+import levi9.UserService.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(value = "/user", produces = MediaType.APPLICATION_JSON_VALUE)
+public class UserController {
+    private final UserService userService;
+
+    @Autowired
+    public UserController(UserService userService) {
+        this.userService = userService;
+    }
+
+    @PostMapping("/register")
+    public ResponseEntity<Object> register(@RequestBody UserDto newUser) {
```

Comment:
After custom exceptions are implemented, return type here can be `ResponseEntity<UserDto>`